### DISCUSSION TITLE
Does the website still work? Getting Twitter API error

### DISCUSSION THREAD

**Discussion by youcrew** [omaha-consulting/winstall#157] [General]
Every time I go to a pack I get there error below. Even when logged into twitter. 

```
Sorry! We could not load this pack.
Recieved error: You currently have access to a subset of Twitter API v2 endpoints and limited v1.1 endpoints (e.g. media post, oauth) only. If you need access to this endpoint, you may need a different access level. You can learn more here: https://developer.twitter.com/en/portal/product
```

**Comment by mherrmann** [comment]
We're running into Twitter rate limits. I'll be happy to merge a PR that fixes the issue.

**Reply by youcrew** [reply]
Thank makes sense, I was wondering if Twitter changed more API rules that harmed the site. Thank you for the response! The merging of a pull request is a bit beyond my knowledge,  as I'm sure you can tell. But thank you anyways!
